Kudu.Contracts/IEnvironment.cs

```diff
@@ -13,6 +13,11 @@ public interface IEnvironment
         string SSHKeyPath { get; }
         string TempPath { get; }
         string ScriptPath { get; }
+
+        /// <summary>
+        /// Path to Kudu SCM site binary folder
```

Comment:
Always good to put a sample value in comment to help clarify (like with `LogFilesPath` below). e.g. is this a physical or virtual path?